tools/public_api_guard/material/core.d.ts

```diff
@@ -197,8 +197,7 @@ export declare const MAT_OPTION_PARENT_COMPONENT: InjectionToken<MatOptionParent
 export declare const MAT_RIPPLE_GLOBAL_OPTIONS: InjectionToken<RippleGlobalOptions>;
 
 export declare class MatCommonModule {
-    constructor(sanityChecks: any, _hammerLoader?: HammerLoader | undefined);
-    _checkHammerIsAvailable(): void;
+    constructor(sanityChecks: any);
```

Comment:
This looks like it _could_ be a breaking change if someone manually passes a `HammerLoader`. I know that this is a rare edge case, but better to talk about it than accidentally missing it.

I'm happy leaving it like that, but could also see having a fake parameter like you did in some other PR (with injecting `ElementRef`)

Reply:
Hmm, good point. My thinking when removing it was that it was optional already.

Reply:
Of all the things people could manually instantiate or extend, `MatCommonModule` would be the strangest. I'm okay just removing this.

Reply:
Yeah, I just wanted to raise awareness. As mentioned above, I'm fine with that either.